### PR TITLE
Initial Runtime

### PR DESCRIPTION
**Pull request by shawntabrizi**
This PR begins work on runtime for the DropIt chain.

- Set Existential Deposit to 0
- Set Various Deposits in Assets Pallet to 0
- Set Multisig Deposits to 0
- Assigns Relay Chain Superuser access to DropIt
- Increases MaxConsumers to 1024
- Adds NFTs Pallet

### CODE REVIEW

File: runtime/mainnet/src/lib.rs [hunk 1]
```diff
@@ -317,6 +323,7 @@ impl frame_system::Config for Runtime {
 	type SS58Prefix = SS58Prefix;
 	/// The action to take on a Runtime Upgrade
 	type OnSetCode = cumulus_pallet_parachain_system::ParachainSetCode<Self>;
+	/// The maximum number of consumers allowed on a single account.
 	type MaxConsumers = frame_support::traits::ConstU32<16>;
```
Comment (joepetrowski) [inline runtime/mainnet/src/lib.rs:327]: In practice this is the number of assets an account can hold without additional deposit. Maybe take the chains off and make it like 256?

Comment (shawntabrizi) [reply runtime/mainnet/src/lib.rs:327]: yeah, i mean with the philosophy of this chain, we should set it to u32 max.

File: runtime/mainnet/src/lib.rs [hunk 2]
```diff
@@ -600,6 +601,7 @@ impl Contains<RuntimeCall> for SafeModeWhitelistedCalls {
 			RuntimeCall::System(_)
 			| RuntimeCall::SafeMode(_)
 			| RuntimeCall::TxPause(_)
+			// balance transfers are allowed
 			| RuntimeCall::Balances(_) => true,
```
Comment (kianenigma) [inline runtime/mainnet/src/lib.rs:605]: not sure why this is permitted even in the template, if in safe mode, we want to allow transfers? that seems to defy the whole purpose of safe mode.

Comment (shawntabrizi) [reply runtime/mainnet/src/lib.rs:638]: good point, let's remove it

Comment (shawntabrizi) [reply runtime/mainnet/src/lib.rs:638]: Same comment about:

```
/// Calls that cannot be paused by the tx-pause pallet.
pub struct TxPauseWhitelistedCalls;
/// Whitelist `Balances::transfer_keep_alive`, all others are pauseable.
impl Contains<RuntimeCallNameOf<Runtime>> for TxPauseWhitelistedCalls {
	fn contains(full_name: &RuntimeCallNameOf<Runtime>) -> bool {
		match (full_name.0.as_slice(), full_name.1.as_slice()) {
			(b"Balances", b"transfer_keep_alive") => true,
			_ => false,
		}
	}
}
```

**Pull request merged**